pages/integrations/inputs/okta_input.rst

```diff
@@ -0,0 +1,42 @@
+.. _okta_input:
+
+*********************
+Okta Log Events Input
+*********************
+
+.. attention:: This is a Graylog Enterprise feature and is only available since Graylog version 3.3. A valid Graylog Enterprise license is required.
+
+
+This input allows Graylog to pull event logs from your Okta domain. For an example of an Okta Log Event object see: https://developer.okta.com/docs/reference/api/system-log/#logevent-object
+
+For this input plugin to function as expected, the following items must be supplied in the input configuration:
+
+1) Domain name
+    Your Okta Domain (also known as your Okta URL). You can copy your domain from the Okta Developer Console.
+    For additional information see: https://developer.okta.com/docs/guides/find-your-domain/overview/
+
+2) API key
+    The token which will be used to authenticate Graylog's requests to Okta.
+    You can create an API token on the Okta Developer Console.
+    For additional information see: https://developer.okta.com/docs/guides/create-an-api-token/-/overview/
+
+3) Pull log events since
+    The lower time bound of the Okta log events and determines how much historical data Graylog will pull from Okta when the Input starts.
+    If not provided, 1 polling interval of historical data will be pulled.  Must be a timestamp in ISO-8601 format."
+
+4) Polling interval
+    Determines how often Graylog will poll for new data stored in Okta. Cannot be smaller than 5 seconds.
+
+5) Keyword filter (optional)
+    The keyword filter is optional and is used to filter the log events results.
+    If it is provided, then it cannot have more than 10 keywords (space-separated) and keywords cannot be more than 40 characters long.
+
+.. Note:: It is recommended that this input only be run on one designated node. Selecting Global will not ensure that the input is running on master, as well as may result in duplicate data.
```

Comment:
This is better than nothing. But this will be something that came up regular. Will it be able to prevent that more than one input is running? Could we implement something in some way?

Reply:
updated the verbiage. if global is selected, then the input will default to only running on the master node